client/src/components/containers/SetCourseForm.tsx

```diff
@@ -66,13 +58,36 @@ const HereButton = styled.button`
   cursor: pointer;
 `;
 
+interface PublishPosition {
+  lat: number
+  lng: number
+}
+
+interface riderPublishInfo {
```

Comment:
```suggestion
interface RiderPublishInfo {
```
or
```suggestion
interface RiderPublishInfoInterface {
```
밑에 동일한 이름의 변수가 있어서 바꾸면 좋을 것 같습니다.